src/plugins/auto_batch/tests/unit/CMakeLists.txt

```diff
@@ -52,5 +51,6 @@ addIeTargetTest(
         LABELS
             Auto_Batch
 )
+ov_add_version_defines(${OpenVINO_SOURCE_DIR}/src/plugins/auto_batch/src/auto_batch.cpp ${TARGET_NAME} CI_BUILD_NUMBER)
```

Comment:
do we need to pass `CI_BUILD_NUMBER` in this macro? I suppose `ov_add_version_defines` can implicitly add `CI_BUILD_NUMBER`